MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1145 @@
+---
+layout: post
+title: Data Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn about the supported data editors in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in .NET MAUI DataForm (SfDataForm)
```

Comment:
# Data Editors in .NET MAUI DataForm (SfDataForm)

Reply:
title updated